### ISSUE TITLE
How can I retrieve data from localStorage?

### ISSUE THREAD

**Issue by MadReal** [robinvdvleuten/vuex-persistedstate#61]
From what I'm understanding from the documentation and my trials, this plugin reads from the states and sets objects converted into JSON in the localStorage.

Now, if I need to create some logic based on data in the storage, where and how should I do that?
I need to check if the user is authenticated (I'm setting is token id manually outside of the plugin) 

`localStorage.setItem('id_token', token)`

if he is and he had previously added some items to the cart (which get saved normally by the plugin)

`{
    key: 'cart',
    storage: localStorage,
    paths: [
        'cartModule.cart'
    ],
}`

I need to grab those items and make post request with axios. 
Logging the cart "localStorage.getItem('cart')" from my action results in a JSON formatted object, which I could convert but I'm not sure if this is how this plugin is meant to be used.

`{"cartModule":{"cart":[{"id":66,"beat":{"id":3,"name":"Issues","producer":{"display_name":"Test Producer","paypal_em.........`

How should I dynamically set and get the data using the plug in and communicating with my store?
I think it would be nice to have a more descriptive documentation, i find it hard to follow as a newbie.

Thank you!

**Comment by robinvdvleuten** [comment]
@MadReal I think you are missing an understanding of a flux architecture here. I recommend just using normal [Vuex getters](https://vuex.vuejs.org/en/getters.html) to retrieve any "state properties".

**Comment by MadReal** [comment]
Hey @robinvdvleuten. No I know about getters, I don't know about caches and cookies though and I'm trying work with this plug in.

So you're saying I shouldn't be checking the local storage data but the store directly?
Hopefully I'm getting a better understanding, but who's "triggered" first? Is it the store that updates the local storage properties or is it / can it be the other way around?

Thanks

**Comment by robinvdvleuten** [comment]
@MadReal exactly! You are mutating the "state" by using localstorage directly, this way the changes are never detected by Vuex.
